docs: mention Apify Academy in blocking guide & link it to proxy and … (#2156)
apify/crawlee: docs/guides/avoid_blocking.mdx

import PlaywrightFingerprintsOffSource from '!!raw-loader!./avoid_blocking_playwright_fingerprints_off.ts';
import PuppeteerFingerprintsOffSource from '!!raw-loader!./avoid_blocking_puppeteer_fingerprints_off.ts';

> A scraper might get blocked for numerous reasons. Let's narrow it down to two main ones. The first one is a bad or blocked IP address. This topic is covered in the [proxy management guide](./proxy-management). The second reason we will explore more in this guide is [browser fingerprints](https://pixelprivacy.com/resources/browser-fingerprinting/) (or signatures). Check the [Apify Academy anti-scraping course](https://docs.apify.com/academy/anti-scraping) to gain deeper theoretical background around blocking and learn extra tips & tricks.

Reviewer: ```suggestion
A scraper might get blocked for numerous reasons. Let's narrow it down to the two main ones. The first is a bad or blocked IP address. You can learn about this topic in the [proxy management guide](./proxy-management). The second reason is [browser fingerprints](https://pixelprivacy.com/resources/browser-fingerprinting/) (or signatures), which we will explore more in this guide. Check the [Apify Academy anti-scraping course](https://docs.apify.com/academy/anti-scraping) to gain a deeper theoretical understanding of blocking and learn a few tips and tricks.
```
